compilation_scripts/cross_compile.sh

```diff
@@ -24,11 +29,24 @@ if [ -f "$ARCH_SPECIFIC_TOOLCHAIN" ]; then
     source $ARCH_SPECIFIC_TOOLCHAIN
 fi
 
+# Note: maybe these should be set from the toolchain CMake file, but it didn't work
+PYTHON_LIBRARY_ARG=""
+if [ ! -z ${PYTHON_LIBRARY} ]; then
+    PYTHON_LIBRARY_ARG="-DPYTHON_LIBRARY=${PYTHON_LIBRARY}"
```

Comment:
I tried to set these flags from the toolchain.cmake file, but it didn't work...
this is not nice but it works